package.json

```diff
@@ -34,7 +34,7 @@
     "eslint-plugin-react": "7.37.4",
     "file-loader": "6.2.0",
     "jest": "29.7.0",
-    "lint-staged": "15.3.0",
+    "lint-staged": "15.4.1",
```

Comment:
src/scripts/lint.ts(77,33): error TS2693: 'lintStaged' only refers to a type, but is being used as a value here.

Reply:
The issue is fixed here: https://github.com/lint-staged/lint-staged/pull/1512